MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Data handling in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the commit feature in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Commit in .NET MAUI DataForm (SfDataForm)
+
+The data form commits the data and user input in order to update the value in the underlying data object.
+
+## Commit mode
+
+The [CommitMode] determines when the value should be committed to the underlying data object to allow users commit only correct values.
```

Comment:
The [CommitMode] determines when the value should be committed to the underlying data object to allow users to commit only correct values.

Reply:
content changed